mkt/extensions/tests/test_models.py

```diff
@@ -527,3 +562,19 @@ def test_pending(self):
         Extension.objects.create(status=STATUS_NULL)
 
         eq_(list(Extension.objects.pending()), [extension1, extension2])
+
+
+class TestExtensionVersionManager(TestCase):
+    def test_pending(self):
+        extension1 = Extension.objects.create()
+        version1 = ExtensionVersion.objects.create(
+            extension=extension1, status=STATUS_PENDING, version='1.1')
+        extension2 = Extension.objects.create()
+        version2 = ExtensionVersion.objects.create(
+            extension=extension2, status=STATUS_PENDING, version='2.1')
+        ExtensionVersion.objects.create(
+            extension=extension2, status=STATUS_PUBLIC, version='2.2')
+        Extension.objects.create(status=STATUS_PUBLIC)
+        Extension.objects.create(status=STATUS_NULL)
+
+        eq_(list(ExtensionVersion.objects.pending()), [version1, version2])
```

Comment:
Yeah, but doing that I would almost replicate the code in the manager method itself... 

I re-organized it a little more to create fewer objects, that should make it easier on the eyes at least.